xtevent/xtevent.ado

```diff
@@ -230,6 +230,11 @@ program define xtevent, eclass
 		}
 	}
 	
+	*skip the rest of the program if the user indicated not to estimate  
+	loc noestimate=r(noestimate)
+	if "`noestimate'"=="." loc noestimate ""
+	if "`noestimate'"!="" exit 
+			

```

Comment:
@Constantino-Carreto-Romero with this we are skipping the rest of the command, but there may be returns that we want to make below, for example `cmd` or `cmdline`. Can you check what we would like to return and move that so it gets returned even if `noestimate` is active,

This may apply to `_eventols` and `_eventiv` as well, although at first glance I did not see any returns there that might be needed.

Reply:
@jorpppp I made some changes in https://github.com/JMSLab/xtevent/commit/ed0f764adc45a7ca8ddf6fa4d6913b15164fe147 so the program will return some values even if `noestimate` is active. For instance, if running:
``` stata
xtevent y, policyvar(z) window(3) savek(aa, noe)
ereturn list
```
ereturn list will show:
![1](https://user-images.githubusercontent.com/36095845/185433269-42f2575e-13f6-46d5-8ee1-4f8b4de7c05c.png)
Other returns are related to the estimation. I only have doubt about including `e(names)`, which returns the name of the event-time dummies, but it doesn't return all the variables because it only consists of the ones included in the regression.
For comparison, here is the list of returned values if running the following:
```stata
xtevent y, policyvar(z) window(3)
```
![2](https://user-images.githubusercontent.com/36095845/185435951-d1d87570-7759-4ec5-b570-9a795654e1ad.png)

Reply:
We won't include `e(names)` in the outputs when `noestimate` is active.